dynawo/sources/Models/Modelica/Dynawo/Electrical/Wind/WECC/BaseClasses/BaseWT4.mo

```diff
@@ -0,0 +1,102 @@
+within Dynawo.Electrical.Wind.WECC.BaseClasses;
+
+/*
+* Copyright (c) 2023, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+partial model BaseWT4 "WECC Wind Turbine model with a simplified drive train model (dual-mass model), without the plant controller and with a current source as interface with the grid"
```

Comment:
Maybe change the comment so that it appears that this is a base model ?

Reply:
Solved